src/systemd/sd-dhcp-client.h

```diff
@@ -58,9 +58,17 @@ enum {
         SD_DHCP_OPTION_INTERFACE_MTU_AGING_TIMEOUT = 24,
         SD_DHCP_OPTION_INTERFACE_MTU               = 26,
         SD_DHCP_OPTION_BROADCAST                   = 28,
+        /* RFC7844, Windows 10 option */
```

Comment:
You're right, useful to me when doing `grep`, to find all my changes related to the RFC, but i should remove it.